pkg/loader/api_loader_test.go

```diff
@@ -83,8 +73,8 @@ func createRegisterAndRouter() (router.Router, error) {
 	r := createRouter()
 	r.Use(middleware.NewRecovery(web.RecoveryHandler).Handler)
 
-	register := proxy.NewRegister(r, createProxy())
-	proxyRepo, err := createProxyRepo()
+	register := proxy.NewRegister(r, proxy.Params{StatsClient: stats.NewStatsdClient("", "")})
```

Comment:
use `stats.NewClient("memory://", "")` in tests - see https://github.com/hellofresh/stats-go#usage-in-unit-tests for details